scripts/gen_vimdoc.py

```diff
@@ -64,6 +64,22 @@
     sys.exit(1)
 
 
+script_dir = os.path.dirname(os.path.abspath(__file__))
+cmake_file_path = os.path.join(script_dir, '..', 'CMakeLists.txt')
+with open(cmake_file_path, 'r') as cmake_file:
+    cmake_content = cmake_file.read()
+
+api_level_match = re.search(r'set\(NVIM_API_LEVEL (\d+)\)', cmake_content)
+api_prerelease_match = re.search(r'set\(NVIM_API_PRERELEASE (\w+)\)', cmake_content)
+
+if not api_level_match or not api_prerelease_match:
+    print("Could not find NVIM_API_LEVEL or NVIM_API_PRERELEASE in CMakeLists.txt")
```

Comment:
Nit: Consistent quoting.
```suggestion
    print('Could not find NVIM_API_LEVEL or NVIM_API_PRERELEASE in CMakeLists.txt')
```

Reply:
also can this logic go in a function

Reply:
done and done